### PR TITLE
fixed failing all-tests-desc set

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
Fixed failing description in all-tests-executor. The pipeline was executed here and did not fail due to the description setting (was aborted after the desc was set): https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/ppaszki-all-tests/

### CODE REVIEW

File: jobs/integr8ly/ocp3/test-suites/all-tests-executor.yaml [hunk 1]
```diff
@@ -103,12 +103,8 @@
...
             timeout(180) { ansiColor('gnome-terminal') { timestamps {
                 node('cirhos_rhel7') {
-                    String description = "cluster url: ${CLUSTER_URL}<br>testing branch: ${BRANCH}<br>" + 
-                        "evals username: ${EVALS_USERNAME}"
-                    if (currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause')['userId']){
-                        description += "<br>triggered by: " + currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause')['userId']
-                    }
-                    currentBuild.description = description
+                    currentBuild.displayName = "${currentBuild.displayName} ${CLUSTER_URL}"
+                    currentBuild.description = "testing branch: ${BRANCH}<br>evals username: ${EVALS_USERNAME}"
```
Comment (tremes) [inline jobs/integr8ly/ocp3/test-suites/all-tests-executor.yaml]: So here the name is `BRANCH` ? :) It's not `INSTALLATION_BRANCH` right :)

Comment (pawelpaszki) [reply jobs/integr8ly/ocp3/test-suites/all-tests-executor.yaml]: https://github.com/integr8ly/ci-cd/pull/286/files#diff-ab9ffde24a59166ffe8a781a189c070dR83

Comment (pawelpaszki) [reply jobs/integr8ly/ocp3/test-suites/all-tests-executor.yaml]: yes. here the param is `BRANCH`

Comment (trepel) [reply jobs/integr8ly/ocp3/test-suites/all-tests-executor.yaml]: Yes, it is indeed `BRANCH` here.

**Pull request merged**